teleport-node/Dockerfile

```diff
@@ -1,21 +1,25 @@
 # Stage1: build from source
-FROM ghcr.io/cybozu/golang:1.21-jammy AS build
+FROM ghcr.io/cybozu/golang:1.22-jammy AS build
+
+ARG TELEPORT_VERSION=15.1.3
+ARG RUST_VERSION=1.71.1
+ARG WASM_PACK_VERSION=0.12.1
 
-ARG TELEPORT_VERSION=14.3.0
 # Install Nodejs
-ARG NODE_VERSION=18.18.2
+ARG NODE_VERSION=18.19.1
```

Comment:
According to the PR title, node.js version is 21.11.1. Which is correct?

Reply:
sorry, 18.19.1 is right. I revice the title

Reply:
I changed again to 18.19.2